src/core/css/inc/mixins.scss

```diff
@@ -67,3 +69,129 @@
       + 'Please make sure it is defined in `$breakpoints` map.';
   }
 }
+
+/* Arrows */
+$default-arrow-margin: 3px;
+@mixin arrow($margin: $default-arrow-margin) {
+  background: url('~amo/img/icons/arrow.svg') 50% 50% no-repeat;
```

Comment:
So I tried inlining this SVG and it is possible but the loader didn't work right out of the box. I did a gross hack like this (seriously lol):

```jsx
import querystring from 'querystring';
import arrow from 'amo/img/icons/arrow.svg';

this.makeLink({
  page: currentPage - 1,
  currentPage,
  pathname,
  query,
  text: (
    <span>
      <i dangerouslySetInnerHTML={{ __html: /<svg.*<\/svg>/.exec(querystring.unescape(arrow))[0] }} />
      {i18n.gettext('Previous')}
    </span>
  ),
})
```

I then set some styles in the developer tools:

```css
i {
    display: inline-block;
    line-height: 1em;
    margin-right: 6px;
    transform: rotate(0.5turn);
}
```

I much prefer the `<i>` or `<svg>` to including all of this CSS (although I feel this could likely be trimmed down a little). This is going to take some work, and we'll likely want to do something like `<Icon style="arrow" reverse />` for the previous arrow and just `<Icon style="arrow" />` for the next arrow. This can be left until later. I was hoping it would be easy in the single case 😢.